test/IFrameNavigatorTests.ts

```diff
@@ -657,6 +657,71 @@ describe("IFrameNavigator", () => {
       });
     });
 
+    it("should show the default SVG logo when custom LibraryIcon is not passed in", async () => {
```

Comment:
This is weird.  I think instead of making libraryIcon optional. let's make upLink optional.  Chances are, you either want to pass a custom logo/link or you don't, I can't imagine that anyone wants to pass in a link to their own site but keep the NYPL logo.  So the tests can just check for, "if you pass a new upLink in, it shows up" and "if you don't, then the NYPL link and logo shows up" 

(It doesn't matter what the NYPL link is, in non-drb it was linking to this github repo, I think that's ok)

Reply:
@hokei  what should the default NYPL link, title and aria-label information to be if someone doesn’t pass in an uplink? Would it be better to hide the uplink if all of the information isn't provided? Currently it's only checking that there is an upLinkConfig with a url.

Currently, all of the items in UpLinkConfig are optional https://github.com/NYPL-Simplified/webpub-viewer/blob/d16ce729d9ab127eec7884df11fada09b108edd8/src/IFrameNavigator.ts#L149-L154 are you recommending that url, label, ariaLabel and libraryIcon become required while upLink remains optional

Reply:
Oh I was thinking https://github.com/NYPL-Simplified/webpub-viewer/blob/d16ce729d9ab127eec7884df11fada09b108edd8/src/IFrameNavigator.ts#L172 but it seems like it's already optional.  
I think you either have your own configuration, or you don't ... and if you have your own configuration, url/label/ariaLabel/libraryIcon should be required.  (Maybe url can be optional, but honestly I don't think it's a big deal for the person who needs the words to not be a link to make that change when they get to it)  

Does that make sense?

Reply:
Yep, that makes sense 👍 I will update accordingly.